test/meta_store_test.rb

```diff
@@ -15,7 +15,6 @@ def mock_request(uri, opts)
 
       def mock_response(status, headers, body)
         headers ||= {}
-        body = Array(body).compact
```

Comment:
cleanup and since I had to pass in an IO object

On Thu, Feb 11, 2016 at 3:13 AM, Alfredo Amatriain <notifications@github.com
> wrote:

> In test/meta_store_test.rb
> <https://github.com/rtomayko/rack-cache/pull/135#discussion_r52587555>:
>
> > @@ -15,7 +15,6 @@ def mock_request(uri, opts)
> >
> >        def mock_response(status, headers, body)
> >          headers ||= {}
> > -        body = Array(body).compact
>
> Why did you remove this? Cleanup, or is it related to the problem with
> streamable bodies?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtomayko/rack-cache/pull/135/files#r52587555>.
>